routers/web/auth/password.go

```diff
@@ -337,6 +344,8 @@ func MustChangePasswordPost(ctx *context.Context) {
 
 	log.Trace("User updated password: %s", u.Name)
 
+	audit.Record(audit.UserPassword, u, u, u, "Password of user %s changed.", u.Name)
```

Comment:
Wait, that isn't `UserPasswordChanged`?

Reply:
What is your point?

Reply:
The constant should be called `UserPasswordChanged` instead of `UserPassword`

Reply:
Isn't the `Changed` implicit? A potential suffix must be added to
```
UserAuthenticationSource
UserActive
UserRestricted
UserAdmin
UserName
UserPassword
UserVisibility

OrganizationName
OrganizationVisibility

RepositoryName
RepositoryVisibility
RepositoryBranchDefault
```
then.

Reply:
Yep, you are right.
Let's leave it at the current one then.